git_machete/github.py

```diff
@@ -150,9 +150,9 @@ def find_pr_by_base_and_head(base: str, head: str, pull_requests: List[GitHubPul
     return None
 
 
-def find_pr_by_number(no: int, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
+def find_pr_by_number(number: int, pull_requests: List[GitHubPullRequest]) -> Optional[GitHubPullRequest]:
```

Comment:
Probably can be replaced with a call to the aforementioned `find_or_none`

Reply:
Not without changes, `find_or_none` (if extracted) would return PR object which has head equal to base of PR passed to this function. In this case, i would expect to return current PR, not the next one in pr chain.

Reply:
Hmmm? how about sth like `find_or_none(lambda pr: number == pr.number, pull_requests)`? can't see how that's limited to _returning PR object which has head equal to base of PR passed to this function_